File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.java


import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    private static String input;
    private static int[] count = new int[26];
    private static int MINUS = 97;

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        input = br.readLine();

        for(int i = 0; i < input.length(); i++) {

            int cur = input.charAt(i)-MINUS;
            count[cur]++;
        }

        for(int i = 0; i < count.length; i++) {
            System.out.print(count[i] + " ");
        }
        System.out.println();
    }
}

File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.py
'''
ord() : 주어진 문자의 Unicode 코드 포인트를 반환하는 파이썬 내장 함수
ord('a') = 97
'''

input_string = input()

count = [0] * 26

for char in input_string:
    count[ord(char) - ord('a')] += 1

print(" ".join(map(str, count)))